src/parser/mod.rs

```diff
@@ -230,10 +232,11 @@ parser!(struct_inline_body _loc -> 'b ast::TypeDefinitionBody<'b> =
     TypeDefinitionBody::Struct(fields)
 );
 
-parser!(import loc =
+parser!(import_defs loc =
```

Comment:
Lets keep the old `import` name, the similarity between `import_defs` and `imported_defs` tripped me up while reading this PR